api/test_resources/boxes.json

```diff
@@ -581,11 +520,17 @@
         "route": "/v2/applications/:application-id/boxes"
       },
       "response": {
-        "statusCode": 404,
-        "body": "{\"message\":\"no application found for application-id: round=8, appid=9223372036854775807\"}\n"
+        "statusCode": 200,
+        "body": "{\"application-id\":9223372036854775807,\"boxes\":[]}\n"
```

Comment:
No longer returning 404 when not finding boxes for an app, even when the app does not exist